docs/Readme.md

```diff
@@ -14,29 +14,36 @@ If you would like to see more examples of field data plug-ins, please visit our
 
 ## Change Log
 
+### AQTS 2020.3 - Framework version 2.10
+- Added support for `GageZeroFlowActivity` activities to record the gage height at zero flow.
+- Added support for `OtherDischargeSection` channel measurements which can be used to record channel measurements measured against any discharge monitoring method.
+- Added support for `EngineeredStructureDischarge` channel measurements which can be used to record the discharge from a physical structure such as a weir or flume.
+- Added support for `VolumetricDischarge` channel measurements which can be used to record the discharge based on an observation of the amount of time it takes for the stream to fill a container of known volume.
+- `ManualGaugingDischargeSection.DischargeMethod` is now optional and will no longer default to `MidSection` if omitted.
+
 ### AQTS 2020.2 - Framework version 2.9
-- Added optional NumberOfVerticals property to ManualGaugingDischargeSection. This property can be used to specify the number of verticals a manual gauging activity has when there are no verticals to be imported. The NumberOfVerticals must be `null` or match the vertical count when the activity has verticals.
-- Added optional MeasurementTime property to DischargeActivity. MeasurementTime is set to half way between the Start and End of the MeasurementPeriod when `null`.
-- Added the `Dictionary<string, string> GetPluginConfigurations()` method to the IFieldDataResultsAppender interface. This method can be used to retrieve configuration settings for each installed plugin. The `Settings` page of the System Config app can be used to change plugin settings in the `FieldDataPluginConfig-{PluginName}` group.
+- Added optional `NumberOfVerticals` property to `ManualGaugingDischargeSection`. This property can be used to specify the number of verticals a manual gauging activity has when there are no verticals to be imported. The `NumberOfVerticals` must be `null` or match the vertical count when the activity has verticals.
```

Comment:
The rest of the changes in this readme are just surrounding some properties in backticks so that they render a bit better from Markdown.